src/odemis/driver/smaract.py

```diff
@@ -415,46 +503,47 @@ def GetMoveStatus(self):
         """
         Gets the move status from the controller.
         Returns:
-            SmarPodDLL.SMARPOD_MOVING is returned if moving
-            SmarPodDLL.SMARPOD_STOPPED when stopped
-            SmarPodDLL.SMARPOD_HOLDING when holding between moves
-            SmarPodDLL.SMARPOD_CALIBRATING when calibrating
-            SmarPodDLL.SMARPOD_REFERENCING when referencing
-            SmarPodDLL.SMARPOD_STANDBY
+            SmarPodDLL.MOVING is returned if moving
+            SmarPodDLL.STOPPED when stopped
+            SmarPodDLL.HOLDING when holding between moves
+            SmarPodDLL.CALIBRATING when calibrating
+            SmarPodDLL.REFERENCING when referencing
+            SmarPodDLL.STANDBY
         """
         status = c_uint()
         self.core.Smarpod_GetMoveStatus(self._id, byref(status))
-        return status
+        return status.value
 
     def Move(self, pos, hold_time=0, block=False):
         """
-        Move to pose command.
+        Move to pose command. It is possible and safe to call it if a previous
+           movement that has been started, called in non-blocking mode.
```

Comment:
This sentence is a bit confusing, should it be:
`It is possible and safe to call it if a previous movement has already been started. Called in non-blocking mode.`
or:
`It is possible and safe to call it if a previous movement has been called in non-blocking mode.`?